typings/primordials.d.ts

```diff
@@ -330,7 +330,7 @@ declare namespace primordials {
   export const ObjectEntries: typeof Object.entries
   export const ObjectFromEntries: typeof Object.fromEntries
   export const ObjectValues: typeof Object.values
-  export const ObjectPrototypeHasOwnProperty: UncurryThis<typeof Object.prototype.hasOwnProperty>
+  export const ObjectHasOwn: typeof Object.hasOwn
```

Comment:
Not sure about this change, both technically exist. In any case, `ObjectHasOwn` should ideally appear before `ObjectValues`, in the same order they are listed in the spec.

Reply:
ObjectPrototypeHasOwnProperty will likely have an inherent slowdown due to the bind in UncurryThis, so preferring ObjectHasOwn makes sense to me

Reply:
I'm talking about this specific change, the one in `typings/primordials.d.ts`, not about the whole PR.

Reply:
This was my intention.

Reply:
I replaced to discourage people from reusing it.

Reply:
In any case, `ObjectHasOwn` should ideally appear before `ObjectValues`, in the same order they are listed in the spec.